src/server/routes/api/user/files/index.ts

```diff
@@ -70,7 +70,7 @@ export default fastifyPlugin(
 
         if (!user) return res.notFound('User not found');
 
-        const perpage = Number(req.query.perpage || '9');
+        const perpage = Number(req.query.perpage || '30');
```

Comment:
I think keeping this at 9, 12, or 15 would be better, it will add 2 more rows and in the future we can plan to add a per-page dropdown on the gallery view like how the table view has.